Lifestyle/todo.30s.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+#
+# Track todos in a simple ~/.todo file
+#
+
+todo_file="$HOME/.todo"
+count=$(wc -l $todo_file | awk '{print $1}')
```

Comment:
```sh
In foo.sh line 7:
count=$(wc -l $todo_file | awk '{print $1}')
              ^-- SC2086: Double quote to prevent globbing and word splitting.
```

Before:
```sh
count=$(wc -l $todo_file | awk '{print $1}')
```

After:
```sh
count=$(wc -l "$todo_file" | awk '{print $1}')
```